ui/packages/audio/src/Audio.svelte

```diff
@@ -57,35 +65,47 @@
 	}
 
 	async function prepare_audio() {
+		await register(await connect());
 		const stream = await navigator.mediaDevices.getUserMedia({ audio: true });
-		recorder = new MediaRecorder(stream);
-
-		recorder.addEventListener("dataavailable", (event) => {
-			audio_chunks.push(event.data);
-		});
-
-		recorder.addEventListener("stop", async () => {
-			if (!streaming) {
-				recording = false;
+		recorder = new MediaRecorder(stream, { mimeType: "audio/wav" });
```

Comment:
These new dependencies are pretty big, do we know how big the treeshaken module is (as we are only importing parts of it)? I'm guessing it is still quite sizable, given what it is doing.

If it is still quite big, it would be good if we only load them when streaming is enabled (via dynamic imports). Happy to help add this functionality if we decide to go this route.

Reply:
I can imagine it would be also sufficient to just use the necessary implementations from this dependency

Reply:
Vendoring in the code would put us on the hook for maintenance which probably isn't ideal as this is not anyone on the team's area of expertise as far as I'm aware.

Reply:
Thats a good point